src/server/ua_services_nodemanagement.c

```diff
@@ -4,6 +4,8 @@
 
 #include "ua_server_internal.h"
 #include "ua_services.h"
+#include "ua_types_generated.h"
+#include "ua_types_generated_handling.h"
```

Comment:
These includes are needed afaik but can be omitted, since they are included in the other header files, that are included here. I can remove them.